beanie/odm/utils/init.py

```diff
@@ -76,7 +78,7 @@ def __init__(
 
         :param database: AsyncIOMotorDatabase - motor database instance
         :param connection_string: str - MongoDB connection string
-        :param document_models: List[Union[Type[DocType], str]] - model classes
+        :param document_models: List[Union[Type[DocType], Type[UnionDocType] str]] - model classes
```

Comment:
```suggestion
        :param document_models: List[Union[Type[DocType], Type[UnionDocType], str]] - model classes
```
Missing a comma here.